### PR TITLE
@ #463 | should override config from teracy dev entry

### PR DESCRIPTION
**Pull request by hieptranquoc**
@hoatle please check this PR! Thank you

connect #463

### CODE REVIEW

**Comment by hoatle** [comment]
@hieptranquoc please follow commit message convention pattern https://github.com/teracyhq/dev/commits/develop

**Comment by hoatle** [comment]
@ #number | message

and make sure to configure your git user matching the github account so that they should match.

**Comment by hoatle** [comment]
focus and make sure to follow the conventions

File: lib/teracy-dev/settings/manager.rb [hunk 1]
```diff
@@ -67,7 +67,15 @@ def build_extensions_settings(extensions)
           next if extension['enabled'] != true
           lookup_path = File.join(TeracyDev::BASE_DIR, extension['path']['lookup'] ||= DEFAULT_EXTESION_LOOKUP_PATH)
           path = File.join(lookup_path, extension['path']['extension'])
-          extensions_settings << Util.build_settings_from(File.join(path, 'config_default.yaml'))
+          if File.exist? "#{path}/config.yaml"
+            extensions_settings << Util.load_yaml_file(File.join(path, 'config.yaml'))
+          else
```
Comment (datphan) [inline lib/teracy-dev/settings/manager.rb]: use `elsif`

File: lib/teracy-dev/settings/manager.rb [hunk 2]
```diff
@@ -67,7 +67,15 @@ def build_extensions_settings(extensions)
...
+            if File.exist? "#{path}/config_default.yaml"
+              @logger.warn("Can not find config.yaml file in #{path}, use config_default.yaml instead. In the future, we will stop using config_default.yaml")
+              extensions_settings << Util.load_yaml_file(File.join(path, 'config_default.yaml'))
```
Comment (datphan) [inline lib/teracy-dev/settings/manager.rb]: we still use `build_settings_from` to load `config_override.yaml`, remove `else`

**Comment by hoatle** [comment]
@hieptranquoc make sure to update the PR and ask for reviewing after that

**Comment by hieptranquoc** [comment]
@datphan @hoatle I updated PR, please re-check

**Review by datphan — APPROVED** [review]

**Review by hoatle — APPROVED** [review]

**Pull request merged**